### PR TITLE
Add new Button component and tests

### PR DESCRIPTION
**Pull request by heinburger**
I did not update package json, but maybe we can continue to add to master until we have what we want for a new manual release (of breaking changes)?

### CODE REVIEW

File: src/Button/Button.js [hunk 1]
```diff
@@ -1,121 +1,66 @@
...
+    font-size: 16px;
+    padding: 11px 30px;
+  `}
+`
+
+const Button = props => <StyledButton {...props} />
+
+Button.defaultProps = {
+  color: colors.blueBayoux,
+  borderOnly: false
```
Comment (menor) [inline src/Button/Button.js]: What about changing `borderOnly` to `hollow`

File: src/variables.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ export const colors = {
 }
 
 export const breakpoints = {
+  // we should name these better perhaps?
```
Comment (menor) [inline src/variables.js]: yes good idea

**Comment by heinburger** [comment]
@menor - I changed the prop to hollow as per your suggestion. It should be ready for merge now

**Pull request merged**